pycrtm.f90

```diff
@@ -703,22 +725,22 @@ END SUBROUTINE aerosols_and_clouds_on
   SUBROUTINE set_emissivity(options, n, use_passed_emissivity )
     USE crtm_MODULE
     IMPLICIT NONE
-    type(crtm_options_type), INTENT(INOUT) :: options(1)
+    type(crtm_options_type), INTENT(INOUT) :: options(:)
```

Comment:
Could you capitalize the TYPE?

Reply:
Sorry, I've been really distracted this past week. Yep, fixed the style, and another spot where it was lowercase instead of uppercase. Also updated the README.